independent-publisher-connectors/OK dokument/apiDefinition.swagger.json

```diff
@@ -0,0 +1,681 @@
+                  "type": "string",
+                  "description": "message",
+                  "title": "message",
+                  "x-ms-visibility": "important"
+                },
+                "result": {
+                  "type": "string",
+                  "description": "result",
+                  "title": "",
+                  "x-ms-visibility": "internal"
+                },
+                "statusCode": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "statusCode",
+                  "title": "",
+                  "x-ms-visibility": "internal"
+                },
+                "success": {
+                  "type": "boolean",
+                  "description": "success",
+                  "title": "success",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "x-ms-visibility": "internal"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Signature Request",
+        "description": "Signature Request",
+        "operationId": "SignatureRequest",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "description": "Document ID from Document Upload action",
+                  "title": "Document ID",
+                  "x-ms-visibility": "important"
+                },
+                "changeState": {
+                  "type": "string",
+                  "description": "changeState",
+                  "title": "changeState",
+                  "default": "tosign",
+                  "x-ms-visibility": "internal"
+                },
+                "allowedFields": {
+                  "type": "object",
+                  "properties": {
+                    "SIGNATURE": {
+                      "type": "string",
+                      "description": "Signature fields in your document",
+                      "title": "Signature fields",
+                      "x-ms-visibility": "advanced"
+                    }
+                  },
+                  "description": "allowedFields"
+                }
+              },
+              "required": [
+                "id",
+                "allowedFields",
+                "changeState"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/document/{Document ID}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {}

```

Comment:
@SoliteaSlovensko empty response schemas are not allowed, please refer to [this example](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses) and [these instructions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses).